storage/local/storage.go

```diff
@@ -770,7 +769,9 @@ func (s *memorySeriesStorage) cycleThroughArchivedFingerprints() chan clientmode
 				}
 				s.waitForNextFP(len(archivedFPs))
 			}
-			glog.Infof("Completed maintenance sweep through archived fingerprints in %v.", time.Since(begin))
+			if len(archivedFPs) > 0 {
+				glog.Infof("Completed maintenance sweep through archived fingerprints in %v.", time.Since(begin))
```

Comment:
Might be interesting to also put the length in there:

    glog.Infof("Completed maintenance sweep through %d archived fingerprints in %v.", len(archivedFPs), time.Since(begin))